app/navigation/Navigator.scala

```diff
@@ -38,7 +38,7 @@ class Navigator @Inject() () {
 
     case CanSealsBeReadPage    => ua => routes.AreAnySealsBrokenController.onPageLoad(ua.id, NormalMode)
     case AreAnySealsBrokenPage => ua => routes.UnloadingFindingsController.onPageLoad(ua.id)
-    case UnloadingCommentsPage => ua => routes.CheckYourAnswersController.onPageLoad(ua.id)
+    case UnloadingCommentsPage => ua => routes.UnloadingFindingsController.onPageLoad(ua.id)
```

Comment:
Probably better to revert this. Our navigation is still based off the MVP. We haven't picked up any navigation work yet. This change could well break journey tests.

Reply:
Sorry, this should've been reverted! It was added for testing purposes 👍